### PR TITLE
[bench] Update bench script with better urls and more info

### PR DESCRIPTION
**Pull request by JasonGross**
**Kind:** infrastructure

This fixes some incorrect URLs in the console output, fixes the links in the comment that coqbot will (fingers crossed) eventually post with benchmarking results, as well as adding a bit more convenience info to that comment.

### CODE REVIEW

File: dev/bench/gitlab.sh [hunk 1]
```diff
@@ -70,8 +74,9 @@ else
     exit 1
 fi
 
-mkdir -p "_bench"
-working_dir="$PWD/_bench"
+bench_dirname="_bench"
+mkdir -p "${bench_dirname}"
```
Comment (SkySkimmer) [inline dev/bench/gitlab.sh:78]: No need for curlies here
```suggestion
mkdir -p "$bench_dirname"
```
and other occurences

Comment (ppedrot) [reply dev/bench/gitlab.sh:78]: Does it hurt though?

Comment (SkySkimmer) [reply dev/bench/gitlab.sh:78]: meh

**Review by ppedrot — APPROVED** [review]

**Comment by ppedrot** [comment]
Will merge soon.

**Pull request merged**